README.md

```diff
@@ -410,6 +412,22 @@ running 0 tests
 test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured
 ```
 
+#### API Property-Based tests
+
+Additionally, the api contains a set of property-based tests. To run these, run CMAKE from a directory within the `api/tests` directory, passing in a flag with the vehicle you want to run the tests for. For example: 
+
+```
+cd api/tests
+mkdir build
+cd build
+cmake .. -DKIA_SOUL=ON
+```
+
+The tests can then be run using `make run-api-property-tests`. For the tests to run, you must have a vcan interface initialized. The test folder contains a script `initialize_vcan.sh` which can be used to set it up automatically.
+
+These tests run against the 1.17.0 Rust toolchain, so it will need to be installed via **rustup** if it isn't already. 
```

Comment:
Can these two lines be moved up above to be below the `Rust installation guide` section? I think it's easy to miss this.

Reply:
881472f